open-metadata-implementation/adapters/open-connectors/rest-client-connectors/spring-rest-client-connector/pom.xml

```diff
@@ -52,6 +52,11 @@
             <groupId>org.codehaus.plexus</groupId>
             <artifactId>plexus-utils</artifactId>
         </dependency>
+
+        <dependency>
```

Comment:
This dependency is pulling in many things - 100s of classes, jetty etc. Since this is client code this seems unnecessary. Is there a more precise dependency that could be used for any needed classes? https://mvnrepository.com/artifact/org.springframework.boot

May be worth doing a maven dependency:tree before and after to see transitive dependencies also, as that would be on top of the classes added here?

Reply:
This dependence is already present there (https://mvnrepository.com/artifact/org.springframework/spring-web/5.2.5.RELEASE) and, in theory, carries all the necessary functionality for this change. So this addiction is not necessary.

Reply:
May be use:
`      DefaultUriBuilderFactory builderFactory = new DefaultUriBuilderFactory();
        builderFactory.setEncodingMode(DefaultUriBuilderFactory.EncodingMode.VALUES_ONLY);
        RestTemplate restTemplate = new RestTemplate();
        restTemplate.setUriTemplateHandler(builderFactory);`

Then there will be no need for spring-boot

Reply:
Many thanks @planetf1 and @immus.

I've made the above change and removed the dependency on spring-boot. That is much nicer :-)

Reply:
We are of course using restTemplate which is deprecated - https://docs.spring.io/spring-framework/docs/current/javadoc-api/index.html?org/springframework/web/client/RestTemplate.html though there is an issue open to track at https://github.com/odpi/egeria/issues/1087

So that IMO is totally out of scope here, but just a reminder.

The new solution makes me happy :-)